corehq/apps/app_manager/_design/views/saved_app/map.js

```diff
@@ -20,7 +20,8 @@ function(doc){
             comment_from: doc.comment_from,
             is_released: doc.is_released,
             case_sharing: doc.case_sharing,
-            build_profiles: doc.build_profiles
+            build_profiles: doc.build_profiles,
+            vellum_case_management: !!doc.vellum_case_management
```

Comment:
@orangejenny reached out to me about this Friday. I said (and I still believe this) that with the app db split out, this is actually a pretty quick operation (i.e. <1h; I recall it being in the minutes but I could be wrong). I wouldn't be too worried about it!